package.json

```diff
@@ -44,12 +44,8 @@
     "storybook": "storybook dev -p 6006"
   },
   "devDependencies": {
-    "@storybook/manager": "^8.0.0-rc.2",
-    "@storybook/manager-api": "^8.0.0-rc.2",
-    "@storybook/preview": "^8.0.0-rc.2",
-    "@storybook/preview-api": "^8.0.0-rc.2",
-    "@storybook/react": "^8.0.0-rc.2",
```

Comment:
In this connection, it is no longer needed as a result of replacing it with react-vite, so it has been removed.
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#moving-from-renderer-based-to-framework-based-configuration